src/adapters/logger.ts

```diff
@@ -1,7 +1,11 @@
 import pino from 'pino'
 
 export const logger = pino({
-  transport: {
-    target: 'pino-pretty'
-  }
+  ...(process.env.LOGGER_TRANSPORT
+    ? {
+        transport: {
+          target: process.env.LOGGER_TRANSPORT
```

Comment:
would be nice to have a default transport to pino-pretty if LOGGER_TRANSPORT is not defined

Reply:
If I'm not wrong, by default Pino uses stdout as the default transport (as per right now).
We are using a different transport in our source code because we are ingesting logs through OpenTelemetry (which could be considered a standard) as it prefers JSON format (similarly to datadog and other products).
I could change the code tomorrow to provide JSON as the default if you want.